packages/realm-server/tests/realm-endpoints/lint-test.ts

```diff
@@ -42,130 +54,821 @@ module(`realm-endpoints/${basename(__filename)}`, function () {
+          );
+        }
+      });
+    });
+
+    module('Prettier Integration Tests', function () {
+      test('applies prettier formatting after eslint fixes', async function (assert) {
+        let response = await request
+          .post('/_lint')
+          .set(
+            'Authorization',
+            `Bearer ${createJWT(testRealm, 'john', ['read', 'write'])}`,
+          )
+          .set('X-HTTP-Method-Override', 'QUERY')
+          .set('Accept', 'application/json')
+          .send(`import { CardDef } from 'https://cardstack.com/base/card-api';
+export class MyCard extends CardDef {
+@field name = contains(StringField);
 }
 `);
 
-      assert.strictEqual(response.status, 200, 'HTTP 200 status');
-      let responseJson = JSON.parse(response.text);
-      assert.strictEqual(
-        responseJson.output,
-        `import StringField from 'https://cardstack.com/base/string';
+        assert.strictEqual(response.status, 200, 'HTTP 200 status');
+        let responseJson = JSON.parse(response.text);
+        assert.strictEqual(
+          responseJson.output,
+          `import StringField from 'https://cardstack.com/base/string';
 import { CardDef, field, contains } from 'https://cardstack.com/base/card-api';
 export class MyCard extends CardDef {
   @field name = contains(StringField);
 }
 `,
-      );
-    });
+          'ESLint fixes are applied and prettier formatting is applied',
+        );
+      });
+
+      test('formats GTS template content properly', async function (assert) {
+        let response = await request
+          .post('/_lint')
+          .set(
+            'Authorization',
+            `Bearer ${createJWT(testRealm, 'john', ['read', 'write'])}`,
+          )
+          .set('X-HTTP-Method-Override', 'QUERY')
+          .set('Accept', 'application/json')
+          .send(`import MyComponent from 'somewhere';
+<template>
+<MyComponent @flag={{eq 1 1}} />
+</template>
+`);
+
+        assert.strictEqual(response.status, 200, 'HTTP 200 status');
+        let responseJson = JSON.parse(response.text);
+        assert.strictEqual(
+          responseJson.output,
+          `import { eq } from '@cardstack/boxel-ui/helpers';
+import MyComponent from 'somewhere';
+<template>
+  <MyComponent @flag={{eq 1 1}} />
+</template>
+`,
+          'GTS template content is properly formatted',
+        );
+      });
+
+      test('handles mixed JavaScript and template content', async function (assert) {
+        let response = await request
+          .post('/_lint')
+          .set(
+            'Authorization',
+            `Bearer ${createJWT(testRealm, 'john', ['read', 'write'])}`,
+          )
+          .set('X-HTTP-Method-Override', 'QUERY')
+          .set('Accept', 'application/json')
+          .send(`import { CardDef } from 'https://cardstack.com/base/card-api';
+import MyComponent from 'somewhere';
 
-    test('user can do a lint with duplicate import fix', async function (assert) {
```

Comment:
Is this test gone? I can’t find it by name, was it subsumed into another?

Reply:
I just incorporated it into another test now.